src/clj/vanilla/db/core.clj

```diff
@@ -99,25 +106,27 @@
 ;; run in the repl to create fresh db tables populated accordingly
 ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;
-(defn initialize-database []
+(defn initialize-database
+  [database]
   (do
     ;; Remove any current tables to start fresh
-    (drop-services-table vanilla-db)
-    (drop-layout-table vanilla-db)
-    (drop-users-table vanilla-db)
+    (drop-services-table database)
+    (drop-layout-table database)
+    (drop-users-table database)
 
     ;; Create new empty versions of the tables
-    (create-services-table vanilla-db)
-    (create-layout-table vanilla-db)
-    (create-user-table vanilla-db)
+    (create-services-table database)
+    (create-layout-table database)
+    (create-user-table database)
 
     ;; Bootstrap any data needed in the DB at start
-    (populate-services)))
+    (populate-services database)))
 
 ;;; REPL ME vvvvv ;;;
 (comment
 
-  (initialize-database)
+  (initialize-database vanilla-default)
```

Comment:
use the {:dbtype dbtype :dbname "vanilla_default"} (from line 27) right inside the call as the parameter. that way we don't pollute the production code

Reply:
Would it be worthwhile to create a command-line app that initialized the db instead of having to execute statements in the repl?

Reply:
We should also look to see if there is Leiningen plugin we could leverage